source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,298 @@
+the MongoDB Server manual. 
+
+.. _java-rs-update-operators:
+
+Update Operators
+~~~~~~~~~~~~~~~~
+
+To change a field in a document, MongoDB provides update operators.
+To specify the modification to perform using the update operators,
+create an update document. To facilitate the creation of update documents, the driver
+provides the ``Updates`` helper class.
+
+.. important::
+
+   The ``id`` field is immutable, so you cannot change the value of the
+   ``id`` field in a document.
+
+To learn more about update operators, see
+:manual:`Update Operators </reference/operator/update/>` in the Server manual.
+
+Update One Document
+-------------------
+
+To update a single document from a MongoDB collection, call the ``updateOne()``
+method and pass your query filter as the first argument. Then, pass your
+update operators as the second argument to the ``updateOne()`` method. After you call the
+``updateOne()`` method, chain the ``subscribe()`` method and pass a subscriber
+as an argument.
+
+The following example uses ``Updates.set()`` to update a document the ``name``
+value of a matching document from ``Bagels N Buns`` to ``2 Bagels 2 Buns``:
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber<UpdateResult> updateOneSubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateOne(eq("name", "Bagels N Buns"), set("name", "2
+    Bagels 2 Buns")).subscribe(updateOneSubscriber);
+    updateOneSubscriber.await();
+
+Update Multiple Documents
+-------------------------
+
+To update multiple documents from a MongoDB collection, call the ``updateMany()``
+method and pass your query filter. Then, pass your
+update operators as the second argument to the ``updateMany()`` method. After you call the
+``updateMany()`` method, chain the ``subscribe()`` method and pass a subscriber
+as an argument.
+
+The following example updates all documents with a ``cuisine`` value of
+``Pizza`` to have a ``cuisine`` value of ``Pasta``:
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber<UpdateResult> updateManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateMany(eq("cuisine", "Pizza"), set("cuisine", "Pasta")).subscribe(updateManySubscriber);
+    updateManySubscriber.await();
+
+Customize the Update Operation
+------------------------------
+
+The ``UpdateOptions`` class contains methods that modify
+the behavior of update methods. To use the ``UpdateOptions``
+class, construct a new instance of the class, then call one or more of its methods
+to modify the update operation. You can chain these method calls together.
+To modify the behavior of the update operation, pass the class instance and
+chained method calls as the second argument to the
+``updateOne()`` or ``updateMany()`` method.
+
+You can use the following optional methods in the ``updateOptions`` class
+to modify an update method:
```

Comment:
S: capitalize "updateOptions"; also, I think using "update operation" instead of "update method" helps differentiate the actual operation from the modifying method:
```suggestion
You can use the following optional methods in the ``UpdateOptions`` class
to modify an update operation:
```